src/__tests__/vuetify.js

```diff
@@ -44,3 +39,13 @@ test('renders a Vuetify-powered component', async () => {
     </div>
   `)
 })
+
+test('allows changing props', async () => {
+  const {queryByText, updateProps} = renderWithVuetify(VuetifyDemoComponent)
+
+  expect(queryByText('This is a hint')).toBe(null)
```

Comment:
To make it similar to other examples, I'd suggest using jest-dom expectations

```suggestion
  expect(queryByText('This is a hint')).not.toBeInTheDocument()
```

you'll need to add the import statement, too:

```js
import '@testing-library/jest-dom'
```